backend/coins/btc/account.go

```diff
@@ -367,17 +371,15 @@ func (account *Account) Initialize() error {
 		account.coin.Net(), account.db, theHeaders, account.synchronizer,
 		account.blockchain, account.notifier, account.log)
 
-	// TODO: unified-accounts
-	gapLimits, err := account.gapLimits(signingConfigurations[0])
-	if err != nil {
-		return err
-	}
-
-	account.log.Infof("gap limits: receive=%d, change=%d", gapLimits.Receive, gapLimits.Change)
-
 	for _, signingConfiguration := range signingConfigurations {
 		signingConfiguration := signingConfiguration
 
+		gapLimits, err := account.gapLimits(signingConfiguration)
```

Comment:
This is used only for the chained addresses. I'd move it in the ` else { ... }` below. Otherwise, it would seem like `gapLimits` is needed for single addresses too.

Reply:
done